frontend/tests/features/wheresMyGene/wheresMyGene.test.ts

```diff
@@ -381,7 +381,8 @@ describe("Where's My Gene", () => {
       expect(numGenes).toBeGreaterThan(0);
     });
 
-    test("Cell types with no marker genes display warning", async ({
+    // need to find a tissue, cell type with no marker genes
+    test.skip("Cell types with no marker genes display warning", async ({
```

Comment:
Just wanted to see if there's a plan for fixing this test this week, or if any help is needed to reenable this 😄 Thanks, @atarashansky !